plugins/packageManager/packages.xql

```diff
@@ -244,15 +243,7 @@ declare %private function packages:display($repoURL as xs:anyURI?, $app as eleme
                                         }
                                         </p>,
                                         <div class="changes" style="display: none;">
-                                        { 
-                                            for $change in $app/changelog/change[@version eq $available or packages:is-newer(./@version, $installed)]
-                                            order by $change/@version descending
-                                            return
-                                                <div class="shortTitle">
-                                                    <h3>{$change/@version/string()}</h3>
-                                                    { $change/node() }
-                                                </div>
-                                        }
+                                        { $app/changelog/change[@version = $available]/node() }
```

Comment:
This change appears to be out of scope of the PR.

Reply:
Oops, I tried to exclude that. I have now fixed that.

Reply:
Thanks!